src/App.tsx

```diff
@@ -1,7 +1,36 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+const emptyKey = '';
+const keyUp = 'keyup';
+
+export class App extends React.Component {
+  state = {
+    pressedKey: emptyKey,
+  };
+
+  componentDidMount() {
+    document.addEventListener(keyUp, (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
+    });
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener(keyUp, () => {
+      this.setState({ pressedKey: emptyKey });
+    });
+  }
+
+  render() {
+    const { pressedKey } = this.state;
+
+    const message = pressedKey === emptyKey
```

Comment:
This line can be simplify:

```
const message = !pressedKey
      ? 'Nothing was pressed yet'
      : `The last pressed key is [${pressedKey}]`;
```

Reply:
I reversed this condition, because is easier to understand.